topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorIT.java

```diff
@@ -134,15 +134,16 @@ public void setup(TestContext context) throws Exception {
         kafkaCluster.deleteDataUponShutdown(true);
         kafkaCluster.usingDirectory(Files.createTempDirectory("operator-integration-test").toFile());
 
-        for (int i = 1; i <= 5; i++) {
+        int maxAttempts = 5;
+        for (int i = 1; i <= maxAttempts; i++) {
             try {
                 LOGGER.info("Running command, attempt: {}", i);
                 kafkaCluster.startup();
                 break;
             } catch (Exception ex) {
-                LOGGER.info("Failed to start kafka cluster. Going to retry it. Number of try: {}", i);
+                LOGGER.info("Failed to start kafka cluster. Going to retry it. Number of try: {}/{}", i, maxAttempts);
                 kafkaCluster.shutdown();
-                if (i == 5) {
+                if (i == maxAttempts) {
```

Comment:
I've just made a very similar change in https://github.com/strimzi/strimzi-kafka-operator/pull/1459/commits/93160fdec07eeb51dd36106d8479a8d69dcbc0ea